<?xml version="1.0"?>
<launch>
  <node pkg="tf" type="static_transform_publisher" name="camera_link7" args="0.059288 -0.029290 0.119091 -0.001457 -0.022181 0.924113 -0.381472 panda_link7 camera 100" required="true"/>

  <node pkg="my_aruco" type="gt_pose_node" name="gt_pose_node" output="screen">
    <param name="frame_id" value="panda_link0"/>
    <param name="child_frame_id" value="camera"/>
  </node>
</launch>
